gunicorn/app/base.py

```diff
@@ -35,7 +35,7 @@ def do_load_config(self):
             self.load_default_config()
             self.load_config()
         except Exception as e:
-            print("\nError: %s" % str(e), file=sys.stderr)
+            print(f"\nError: {e}\n", file=sys.stderr)
```

Comment:
Note: additional newline added here; I'll undo that since it's a behaviour change.